overlay/flask_server.py

```diff
@@ -61,6 +68,18 @@ def get_schema():
     return jsonify(schema_json)
 
 
+@app.route("/validation_service/streaming", methods=["GET"])
```

Comment:
How about 
```suggestion
@app.route("/validation_service/streaming_test", methods=["GET"])
```
?

Reply:
I thought of adding the `_test` suffix to the POST request since this is a test and doesn't add functionality to the validation service. I'm fine with either one.